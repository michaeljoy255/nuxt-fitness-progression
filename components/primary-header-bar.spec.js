import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'
import PrimaryHeaderBar from './primary-header-bar'

const localVue = createLocalVue()
let wrapper

beforeEach(() => {
  const vuetify = new Vuetify()
  wrapper = mount(PrimaryHeaderBar, {
    stubs: {},
    propsData: {},
    mocks: {
      $store: {},
      $router: {},
      $route: {},
    },
    localVue,
    vuetify,
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('Component > primary-header-bar', () => {
  test('matches the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
