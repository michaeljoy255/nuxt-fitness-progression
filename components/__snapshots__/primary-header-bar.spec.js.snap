// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > primary-header-bar matches the snapshot 1`] = `
"<header class=\\"v-sheet theme--light v-toolbar v-app-bar v-app-bar--clipped v-app-bar--fixed secondary\\" style=\\"height: 64px; margin-top: 0px; transform: translateY(0px); left: 0px; right: 0px;\\">
  <div class=\\"v-toolbar__content\\" style=\\"height: 64px;\\"><button type=\\"button\\" class=\\"v-app-bar__nav-icon v-btn v-btn--flat v-btn--icon v-btn--round theme--light v-size--default\\"><span class=\\"v-btn__content\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-menu theme--light\\"></i></span></button>
    <div class=\\"v-toolbar__title\\">Fitness LP</div>
  </div>
</header>"
`;
